src/core/qobj.jl

```diff
@@ -70,19 +68,21 @@ julia> _ψ * ψ    # A Bra times a Ket is a scalar
 1.0 + 0.0im
 
 julia> ψ*_ψ     # A Ket times a Bra is an operator
-(3, 3)-element Snowflake.Operator:
-Underlying data Matrix{ComplexF64}:
+(3, 3)-element Snowflake.DenseOperator:
+Underlying data ComplexF64:
```

Comment:
It was decided that it would be better to not expose the datatype used internally, so that changes in implementation would not change user-facing features